Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -986,6 +986,54 @@ namespace Tensile
                 }
             };
 
+            struct GlobalSplitUCheckK
```

Comment:
Just as a matter of naming, I would recommend GlobalSplitUCheckMinK . IMO just a bit more descriptive

Reply:
Hi @cgmillette,
Thanks for you suggestion.
We change our patch according your comment.
Please help to review again.
By the way, _GlobalAccumulation contains condition of GSU > 1.